.generate

```diff
@@ -145,7 +145,6 @@ class disposableHostGenerator():
         {'type': 'list', 'src': 'https://raw.githubusercontent.com/FGRibreau/mailchecker/master/list.txt'},
         {'type': 'json', 'src': 'https://raw.githubusercontent.com/ivolo/disposable-email-domains/master/index.json'},
         {'type': 'list', 'src': 'https://raw.githubusercontent.com/7c/fakefilter/main/txt/data.txt'},
-        {'type': 'list', 'src': 'https://getnada.com/api/v1/domains'},
```

Comment:
`getnada.com` redirects to `inboxes.com`, which uses `https://inboxes.com/api/v2/domain` (JSON format). So maybe instead of removing this one, it can be updated to work with `inboxes.com`?